src/main/java/mekanism/client/gui/GuiChemicalCrystallizer.java

```diff
@@ -48,7 +49,7 @@
 
 	public Gas prevGas;
 
-	public ItemStack renderStack;
+	public @Nullable ItemStack renderStack;
```

Comment:
It isn't used in this project, but it's a public value which could be changed by any other mod or Mekanism Addon.